extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeChannelExtractor.java

```diff
@@ -162,6 +164,24 @@ public String getDescription() throws ParsingException {
         }
     }
 
+    /**
+     * Youtube might show a red warning text for example
+     * "This account has been terminated due to multiple or severe violations of
+     * YouTube's policy prohibiting hate speech."
+     * In that case parsing of the channel information will fail. This error text should
+     * be displayed to the user instead of a cryptic ParsingException.
+     * @return the text as String or null if no warning text could be found
+     */
+    @Override
+    public String obtainErrorMessage() {
+        try {
+            return doc.select("div.yt-alert-message").first().text();
```

Comment:
Please exclude the old browser message. `:not(#old-browser-alert)`

<details><summary>HTML</summary>

``` HTML
<div class="alerts-wrapper">
  <div id="alerts" class="content-alignment">
   <div class="yt-alert yt-alert-default yt-alert-warn  " id="old-browser-alert">
    <div class="yt-alert-icon">
     <span class="icon master-sprite yt-sprite"></span>
    </div>
    <div class="yt-alert-content" role="alert">
     <div class="yt-alert-message" tabindex="0">
       We'll stop supporting this browser soon. For the best experience, please
      <a href="/supported_browsers">update your browser</a>.
     </div>
    </div>
    <div class="yt-alert-buttons"></div>
   </div>
   <div class="yt-alert yt-alert-default yt-alert-error  ">
    <div class="yt-alert-icon">
     <span class="icon master-sprite yt-sprite"></span>
    </div>
    <div class="yt-alert-content" role="alert">
     <div class="yt-alert-message" tabindex="0">
       This account has been terminated due to multiple or severe violations of YouTube's policy prohibiting hate speech.
     </div>
    </div>
    <div class="yt-alert-buttons">
     <button class="yt-uix-button yt-uix-button-size-default yt-uix-button-close close yt-uix-close" type="button" onclick=";return false;" aria-label="Close" data-close-parent-class="yt-alert"><span class="yt-uix-button-content">Close</span></button>
    </div>
   </div>
  </div>
 </div>
```
</details>